### PR TITLE
kinds.ts: Fix connectToPlural() for plugin's models

### PR DESCRIPTION
**Pull request by mareklibra**
This change reflects models provided by plugins.

### CODE REVIEW

**Comment by mareklibra** [comment]
Cc @vojtechszocs

File: frontend/public/kinds.ts [hunk 1]
```diff
@@ -34,7 +33,7 @@ export const connectToPlural: ConnectToPlural = connect((state: State, props: {p
 
   const kindObj = isGroupVersionKind(plural)
     ? state.k8s.getIn(['RESOURCES', 'models']).get(plural)
-    : _.find(k8sModels, model => model.plural === plural);
+    : _.find(allModels().toArray(), model => model.plural === plural);
```
Comment (spadgett) [inline frontend/public/kinds.ts]:
```suggestion
    : allModels().find(model => model.plural === plural);
```

Comment (mareklibra) [reply frontend/public/kinds.ts]: done

**Pull request merged**